driver/userspace_shared.h

```diff
@@ -40,25 +40,25 @@ typedef struct _USER_IN {
     INT             Pid;
     UINT64          DiskSize;
     UINT16          BlockSize;
-} USER_IN, * PUSER_IN;
+} CONNECTION_INFO, * PCONNECTION_INFO;
 
 typedef struct _USER_COMMAND {
```

Comment:
Do we really need a structure? Can we do a typedef? Also, a better name might be "WNBD_COMMAND"

Reply:
As discussed offline:
`
it' usually just better to do sizeof struct instead of something that can hold the ioctl code and also it's easier if we want to add something more instead of just the ioctl code i.e. feed other flags like operational mode.
`

I will change the naming in a different PR since I want to push more changes beside this one.

Closing this PR and opening a new one.